docs/docusaurus/docs/cloud/checkpoints/manage_checkpoints.md

```diff
@@ -81,6 +81,40 @@ To learn more about Checkpoints, see [Checkpoint](/reference/learn/terms/checkpo
 
 4. Click **Run Checkpoint** for the Checkpoint you want to run.
 
+
+## Add a Validation to a Checkpoint
+
+Add Validation data to a Checkpoint to aggregate individual Expectation Suite or Data Source Validations into a single Checkpoint. For more information, see [Add Validation data or Expectation Suites to a Checkpoint](../../oss/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md) in the GX OSS documentaion.
+
+1. In GX Cloud, click **Checkpoints**.
+
+2. Click **Edit Checkpoint** in the **Checkpoints** list for the Checkpoint you want to add the Validation.
+
+3. Copy the code snippet and then close the **Edit Checkpoint** dialog.
+
+4. Paste the the code snippet into Jupyter Notebook and then add the following code block:
+
+    ```python title="Jupyter Notebook"
+        validations = [
+        {
+            "batch_request": {
+                "datasource_name": "your_datasource_name",
+                "data_asset_name": "your_data_asset_name",
+            },
+            "expectation_suite_name": "your.expectation.suite.name",
+        },
+    ]
```

Comment:
This should be pulled from a snippet in an  example script under test.

Reply:
@Rachel-Reverie as discussed, what you see is a temporary solution (I hope). In the future, running code should be unnecessary.